src/components/views/article/Article.spec.js

```diff
@@ -189,33 +205,15 @@ describe('Article component', () => {
     };
 
     const component = mount(
-      <Router>
-        <ArticleComponent {...props} />
-      </Router>,
+      <Provider store={store}>
+        <Router>
+          <ArticleComponent {...props} />
+        </Router>
+      </Provider>,
     );
 
     const relatedArticleContainer = component.find('.related__article__container');
     const articleTitle = relatedArticleContainer.find('#article-title');
     expect(articleTitle.html()).toEqual('<p id="article-title">The Curious Case of Benjamin Buttons</p>');
   });
-  it('should render related articles', () => {
-    const props = {
-      signleArticle: articleWithoutTags,
-      loading: false,
-      error: {},
-      getArticleBySlug: jest.fn(),
-      match: {
-        params: { slug: 'some-slug' },
-      },
-      relatedArticles: [article],
-    };
-    const component = mount(
-      <Router>
-        <ArticleComponent {...props} />
-      </Router>,
-    );
-    const relatedArticleContainer = component.find('.related__article__container');
-    const articleTitle = relatedArticleContainer.find('#article-title');
-    expect(articleTitle.html()).toEqual('<p id="article-title">The Curious Case of Benjamin Buttons</p>');
-  });
 });
```

Comment:
And this, what happened to this block of code?

Reply:
@OdunayoOkebunmi this block of code is still not back

Reply:
There's a duplicate in that particular test suite